tests/version.yaml

```diff
@@ -0,0 +1,64 @@
+image:
+  tag: 2.7
+
+env:
+  PGSCHEMA:
+    value: main
+  PGHOST:
+    value: test-pg-postgresql
+  PGHOST_SLAVE:
+    value: test-pg-postgresql
+  PGPORT:
+    value: '5432'
+  PGPORT_SLAVE:
+    value: '5432'
+  PGDATABASE:
+    value: postgres
+  PGPASSWORD:
+    value: mySuperTestingPassword
+  PGUSER:
+    value: postgres
+  AUTHTKT_SECRET:
+    value: '1234567890123456789012345678901234567890123456789012345678901234'
+
+config:
+  image:
+    repository: camptocamp/geomapfishdemo-config
+    tag: prod-2-7
+  env:
+    VISIBLE_WEB_HOST:
+      value: example.com
+
+alembic:
+  image:
+    tag: 2.7
+  mainStatic: true
+  env:
+    PGSCHEMA:
+      value: main
+    PGHOST:
+      value: test-pg-postgresql
+    PGHOST_SLAVE:
+      value: test-pg-postgresql
+    PGPORT:
+      value: '5432'
+    PGPORT_SLAVE:
+      value: '5432'
+    PGDATABASE:
+      value: postgres
+    PGPASSWORD:
+      value: mySuperTestingPassword
+    PGUSER:
+      value: postgres
```

Comment:
I don't like the need of adding 2 times the `env` section. It might be useful to replace this line:
https://github.com/camptocamp/helm-geoportal/pull/14/files#diff-2024f90f6ef05bf8e3a0cc367802a1eb4eebbea11a52d860c464f5542e0f4afdR19
with an helper function that uses the already defined `Values.env` here, instead of the need of a `Values.alembic.env`. AFAIU they should be the same, so doubling them adds a risk of errors.